src/extensions/extensionInterface.ts

```diff
@@ -294,6 +295,14 @@ type SetWaveformsModeRecordingSelectionAction = {
 
 export type RecordingSelectionAction = SetRecordingSelectionRecordingSelectionAction | SetSelectedElectrodeIdsRecordingSelectionAction | SetVisibleElectrodeIdsRecordingSelectionAction | SetCurrentTimepointRecordingSelectionAction | SetTimeRangeRecordingSelectionAction | SetAmpScaleFactorRecordingSelectionAction | ScaleAmpScaleFactorRecordingSelectionAction | SetCurrentTimepointVelocityRecordingSelectionAction | SetWaveformsModeRecordingSelectionAction
 
+const adjustTimeRangeToIncludeTimepoint = (timeRange: {min: number, max: number}, timepoint: number) => {
```

Comment:
This adjustment is a translation of the time window, not a scaling: we keep the same size window and just move it to center it on the input `timepoint`.

Reply:
yes